controller/lib/core/sensors.h

```diff
@@ -86,8 +104,28 @@ class Sensors {
   // Calibrated average sensor values in a zero state.
   Voltage sensors_zero_vals_[NUM_SENSORS];
 
-  // State related to integrating volume from flow.
+  // Our flow sensors are subject to some zero-point drift, meaning that the
+  // flow we observe is equal to true flow plus an error, and that error
+  // changes over time.
+  //
+  // We currently use a PID controller to estimate this flow error.  We invoke
```

Comment:
Please somewhere in this block of comments say in one sentence, what is the input (measured value and setpoint) and output (controlled value) of the PID.